deployment/ansible/roles/model-my-watershed.rwd/templates/upstart-mmw-rwd.conf.j2

```diff
@@ -0,0 +1,28 @@
+description "Rapid Watershed Delineation"
+
+start on (filesystem and started docker)
+stop on stopping docker
+
+kill timeout 20
+kill signal CONT
+respawn
+
+pre-start script
+  /usr/bin/docker kill rwd || true
+  /usr/bin/docker rm rwd || true
+end script
+
+exec /usr/bin/docker run \
+  --name rwd \
+  --publish {{ rwd_port }}:{{ rwd_port }} \
+  --volume {{ rwd_data_path }}:{{ rwd_data_path }} \
```

Comment:
It looks like Jenkins was able to start this service without encountering an error because of this line so I'm inclined to leave as-is for now.